### PR TITLE
E-1268 feat(utils): add getStartDay and getEndDay utils into date

### PR DESCRIPTION
**Pull request by ifiri**

### CODE REVIEW

File: packages/utils/src/date.js [hunk 1]
```diff
@@ -47,12 +47,26 @@ const toDayBeginInUTC = value => {
   return utcDate;
 };
 
+const getStartDay = day => {
```
Comment (kazurus) [inline packages/utils/src/date.js:50]: types?

Comment (ifiri) [reply packages/utils/src/date.js:50]: for now utils not typed

Comment (dudiq) [reply packages/utils/src/date.js:50]: decalred in `types/date.d.ts`

Comment (ifiri) [reply packages/utils/src/date.js:50]: done

Comment (dudiq) [reply packages/utils/src/date.js:50]: hmm, you can try to copy tsconfig.js from `class` package and fix declares

**Review by kazurus — APPROVED** [review]
@dudiq approved as well

**Pull request merged**